pkg/util/node/node.go

```diff
@@ -177,3 +177,27 @@ func PatchNodeStatus(c clientset.Interface, nodeName types.NodeName, oldNode *v1
 	}
 	return updatedNode, nil
 }
+
+// PatchNodeSpec patches node spec.
```

Comment:
I am not sure it is the correct way to patch a node `Spec`. @kubernetes/sig-node-api-reviews, Could you please  take a look at this.